### PR TITLE
Use the ansible.com nav bar

### LINKED ISSUE #74: Keep ansible.com top-level navigation
- Can you make the header of the docsite exactly the same as current [docs.ansible.com](http://docs.ansible.com/) (with Product, Blog, Community etc....)
- Create a sub header with Ansible core, Ansible ecosystem, Ansiible automatioin platform, Join the communiity
- On that sub header, can you rename Ansible automation platform with Red Hat Ansible Automation Platform

### PR DESCRIPTION
**Pull request by oraNod**
Fixes #74

### CODE REVIEW

**Comment by samccann** [comment]
@gotmax23 valid point. We plan on swapping out these pages for what is on docs.ansible.com  on Tuesday.  Since it's still ansible.com, not our new community website, we want to keep the top header consistent with the rest of ansible.com.

**Comment by samccann** [comment]
We're starting to lose realestate with this double top nav.  Two suggestions (which may be impossible):
1 - can we shrink the amount of black space above/below  the ansible.com banner to make it take up a little less space?
2 - Can we strip Ansible out of all the nav links in the community top nav so the line doesn't wrap?

image

Marking as approved if we can't trim it down a bit. Don't want to hold this up.

**Review by samccann — APPROVED** [review]

**Comment by oraNod** [comment]
@samccann I've reduced the padding on the black ansible.com nav. Unfortunately I think being stuck with that ansible.com nav is just going to make things difficult. Maybe we could work out a left-nav sort of thing but I feel like that would really slow us down and can probably wait for a future optimization.

**Pull request merged**
